RELEASE_NOTES.md

```diff
@@ -24,7 +24,31 @@
 
 ### Base platform component upgrades
```

Comment:
cray-metallb upgraded to 0.14.9
sealed-secrets upgraded to 0.28.0
cray-node-problem-detector v0.8.20
cray-vault-operator v1.22.5
cray-externaldns 0.15.0

Reply:
bitnami-etcd, etcd, ceph, istio, kiali, postgres-operator versions are good.
postgresql version should be 15.2